lms/templates/dates_banner.html

```diff
@@ -74,7 +81,7 @@
                 ${_(' To complete graded assignments as part of this course, you can upgrade today.')}
             % endif
         </div>
-        % if not is_mobile_app:
+        % if not is_mobile_app or is_learning_mfe:
```

Comment:
Do you need all of these `or`s? I would assume that if `is_learning_mfe`, it will also be `not is_mobile_app` so it seems like a redundant check.

Reply:
is_mobile_app checks which view is hit.  the learning mfe hits the mobile view....   its gross i know